static/base.html

```diff
@@ -19,7 +19,10 @@
                     <img src="{{ url_for('static', filename='img/logo.svg') }}" class="me-2" alt="logo">
                     <span class="text-body-emphasis align-self-baseline fs-4">ankerctl</span>
                 </a>
-                <ul class="nav nav-pills align-items-baseline" id="tab" role="tablist">
+                <ul class="nav nav-pills align-items-baseline px-3" id="printBtn" role="tablist">
+                    {% block print_button %}{% endblock %}
```

Comment:
Isn't this a remnant of the "print" button we removed?

Reply:
Yes, I thought I had removed this. I am away from home, feel free to remove it and push to my branch.